ReactCommon/jsi/jsi/test/testlib.cpp

```diff
@@ -394,6 +394,23 @@ TEST_P(JSITest, HostObjectTest) {
                    .getBool());
 }
 
+TEST_P(JSITest, HostObjectProtoTest) {
+  class ProtoHostObject : public HostObject {
+    Value get(Runtime& rt, const PropNameID&) override {
+      return String::createFromAscii(rt, "phoprop");
+    }
+  };
+
+  rt.global().setProperty(
+      rt,
+      "pho",
+      Object::createFromHostObject(rt, std::make_shared<ProtoHostObject>()));
+
+  EXPECT_EQ(
```

Comment:
It looks like this code tests that we can use HostObjects as object prototypes.
Seems legit but I don't understand what's the point to use `Symbol.toPrimitive` here, we can use any symbol here.